styles.css

```diff
@@ -35,6 +35,48 @@ html {
   height: 48px;
 }
 
+header .mobile-menu-overlay {
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  background-color: #3c3a39;
+  padding-top: 18px;
+  transition: all 0.5s ease-out;
+  overflow-x: hidden;
+}
```

Comment:
- [ ] good job fixing the issue the previous CR notified you about, however, when fixing this another bug was created. when loading the application the first time the mobile menu is open even in desktop mode, you can fix this by changing `display: flex;` to `display: none;` in the `mobile-menu-overlay` class. please make sure to check for any other bugs after making this change as well.